nvidia/nvidia-utils/.SRCINFO

```diff
@@ -1,6 +1,6 @@
 pkgbase = nvidia-utils
 	pkgver = 560.35.03
-	pkgrel = 9
+	pkgrel = 16
```

Comment:
Do we need to sync pkgrel too? This is a big jump.

Reply:
Yes, we need to. Otherwise its reported by compare_archpkgver as outdated and vladdy doesnt like that.